apps/android/app/src/main/res/values-de/strings.xml

```diff
@@ -712,38 +712,38 @@
 
   <!-- Group event chat items -->
-  <string name="rcv_group_event_member_added">***sie haben <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> eingeladen.</string>
-  <string name="rcv_group_event_member_connected">verbunden</string>
+  <string name="rcv_group_event_member_added">hat <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> eingeladen.</string>
+  <string name="rcv_group_event_member_connected">beigetreten</string>
   <string name="rcv_group_event_member_left">verlassen</string>
-  <string name="rcv_group_event_changed_member_role">***changed role of %s to %s</string>
-  <string name="rcv_group_event_changed_your_role">***changed your role to %s</string>
-  <string name="rcv_group_event_member_deleted">entfernte <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g>.</string>
-  <string name="rcv_group_event_user_deleted">hat Sie entfernt</string>
-  <string name="rcv_group_event_group_deleted">gruppe gelöscht</string>
-  <string name="rcv_group_event_updated_group_profile">aktualisiertes gruppenprofil</string>
-  <string name="rcv_group_event_invited_via_your_group_link">***invited via your group link</string>
-  <string name="snd_group_event_changed_member_role">***you changed role of %s to %s</string>
-  <string name="snd_group_event_changed_role_for_yourself">***you changed role for yourself to %s</string>
-  <string name="snd_group_event_member_deleted">sie haben <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> entfernt.</string>
-  <string name="snd_group_event_user_left">sie haben verlassen</string>
-  <string name="snd_group_event_group_profile_updated">gruppenprofil aktualisiert</string>
+  <string name="rcv_group_event_changed_member_role">änderte die Rolle von %s auf %s</string>
+  <string name="rcv_group_event_changed_your_role">änderte Ihre Rolle auf %s</string>
+  <string name="rcv_group_event_member_deleted">entfernte <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> aus der Gruppe.</string>
+  <string name="rcv_group_event_user_deleted">hat Sie aus der Gruppe entfernt</string>
+  <string name="rcv_group_event_group_deleted">Gruppe gelöscht</string>
+  <string name="rcv_group_event_updated_group_profile">Aktualisiertes Gruppenprofil</string>
+  <string name="rcv_group_event_invited_via_your_group_link">wurde über Ihren Gruppen-Link eingeladen</string>
+  <string name="snd_group_event_changed_member_role">Sie haben die Rolle von %s auf %s geändert</string>
+  <string name="snd_group_event_changed_role_for_yourself">Sie haben Ihre eigene Rolle auf %s geändert</string>
+  <string name="snd_group_event_member_deleted">entfernt <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> aus der Gruppe.</string>
+  <string name="snd_group_event_user_left">hat die Gruppe verlassen</string>
+  <string name="snd_group_event_group_profile_updated">Gruppenprofil aktualisiert</string>
```

Comment:
same here
```suggestion
  <string name="rcv_group_event_updated_group_profile">aktualisiertes Gruppenprofil</string>
  <string name="rcv_group_event_invited_via_your_group_link">wurde über Ihren Gruppen-Link eingeladen</string>
  <string name="snd_group_event_changed_member_role">sie haben die Rolle von %s auf %s geändert</string>
  <string name="snd_group_event_changed_role_for_yourself">sie haben Ihre eigene Rolle auf %s geändert</string>
  <string name="snd_group_event_member_deleted">entfernt <xliff:g id="member profile" example="alice (Alice)">%1$s</xliff:g> aus der Gruppe.</string>
  <string name="snd_group_event_user_left">hat die Gruppe verlassen</string>
  <string name="snd_group_event_group_profile_updated">gruppenprofil aktualisiert</string>
```

Reply:
also maybe other words shouldn't be capitalized as well, e.g. "aktualisiertes gruppenprofil" - these service messages were intended to be as little intrusive as possible and have no accent